CHANGELOG.md

```diff
@@ -58,22 +89,6 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 -
 
-## [0.1.50] - 2022-04-05
```

Comment:
I didn't catch this in the previous review [0].

I was wondering why this what already in the file when we hadn't release v0.1.50, yet. I think we should be OK now if the Makefile target generated these notes.

[0] https://github.com/ComplianceAsCode/compliance-operator/commit/0dfd6115990a3c6f6b0df39a906e4f97ebca6946